### ISSUE TITLE
Complete URLs in the "Downloads" tab are only copiable after completion (what's being changed since version 127?)

### ISSUE THREAD

**Issue by starchivore** [brave/brave-browser#40026]
### Description

Good Day,

Back in the day of version `126` (i.e. release `1.67.X`) or below, everything available in the `Downloads` tab could still be copied in the middle of downloading or even if any downloads were being canceled. Obviously that's so much more convenient simply because we wouldn't have to wait until those HUGE downloads to be completed when all we ever needed should be grabbing the **_complete_** URLs very quickly.

Unfortunately something should have been changed at some point, and then version `127` (i.e. release `1.68.X`) wouldn't show any **_complete_** URLs at all **_unless and until_** each download is 100% done:

[https://imgur.com/VY4q9Ft](https://imgur.com/VY4q9Ft)
![VY4q9Ft](https://i.imgur.com/VY4q9Ft.png)

For some reasons I could merely copy the domain name (i.e. **_incomplete_** URLs) after cancelling any downloads, while nothing whatsoever could be copied when each download was still in progress:

[https://imgur.com/LaAOwbc](https://imgur.com/LaAOwbc)
![LaAOwbc](https://i.imgur.com/LaAOwbc.png)

Here's yet another example, Microsoft Enterprise Windows Driver Kit is a whopping 17.8GB file and it just doesn't make sense to get the whole thing IMHO:

[https://learn.microsoft.com/en-us/legal/windows/hardware/enterprise-wdk-license-2022](https://learn.microsoft.com/en-us/legal/windows/hardware/enterprise-wdk-license-2022)

BTW, please don't tell us to press F12 and grab those complete URLs under the `Network` tab instead and many thanks.

### Steps to reproduce

1. Start any downloads (everything will behave the same way, obviously NOT limited to specific downloads)
2. Cancel any downloads, for some reasons only the domain name could be copied
3. **_Complete_** URLs could only be copied after each download is 100% done

### Actual result

Only the domain name could be copied, since the introduction of version `127`

### Expected result

**_Complete_** URLs could be copied, just like the way it was back in version `126` or below

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

`Brave` 1.68.128 Chromium: 127.0.6533.73 (Official Build) (64-bit)
`Revision` 800676fd9ede2eb4174bac98c186e53749c7201f
`OS` Windows 11 Version 24H2 (Build 26100.1150)

### Channel information

- [X] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by starchivore** [comment]
[Release v1.67.134 (Chromium 126.0.6478.186)](https://github.com/brave/brave-browser/releases/tag/v1.67.134)

Reverting back to the latest version `126` and everything looked fine, even cancelling a download would still allow the **_complete_** URL to be copied:

[https://imgur.com/QR23BMd](https://imgur.com/QR23BMd)
![QR23BMd](https://i.imgur.com/QR23BMd.png)

Same deal with pausing a similar download:

[https://imgur.com/wK4Rkgc](https://imgur.com/wK4Rkgc)
![wK4Rkgc](https://i.imgur.com/wK4Rkgc.png)

Further proof for changes between version `126` and `127` since we'd like to have something that's more concrete, I tried the earliest nightly release `127.0.6533.17` available and that already exhibited the same kinda "anomaly" as I described before:

[Nightly v1.69.51 (Chromium 127.0.6533.17)](https://github.com/brave/brave-browser/releases/tag/v1.69.51)

In other words, not sure if that were supposed to be more like a "Chromium" thing or just related to specific changes applied to the UI of Brave somewhere?

Thanks again for your help.
